docs/software/dns_servers.md

```diff
@@ -2,14 +2,114 @@
 
 ## DNS Server
 
-!!! warning "Documentation in progress..." 
+- [**Pi-hole - A DNS/Web server that will block ads for any device on your network**](#pi-hole-a-dnsweb-server-that-will-block-ads-for-any-device-on-your-network)
 
-    The links redirect to the old documentation available on [DietPi.com Forum](https://dietpi.com/phpbb/viewtopic.php?f=8&t=5). **We need your help to create and update this documentation ! Thank you** :octicons-heart-16: :octicons-heart-16:
+## Dynamic DNS Clients
 
-- [Pi-hole - A DNS/Web server that will block ads for any device on your network.](https://dietpi.com/phpbb/viewtopic.php?f=8&t=5&start=20#p174)  
+- [**No-IP - Dynamic DNS update client**](#no-ip-dynamic-dns-update-client)
 
-## Dynamic DNS Clients
+??? info "How do I run **DietPi-Software** and install **Optimised software** ?"
+    To install any of the **DietPi Optimised Software** listed below run from the command line:
+    ```
+    dietpi-launcher
```

Comment:
Same as with the other software pages: I suggest to keep it simple and use `dietpi-software` directly only. Probably instead of a collapsible info box, it can then be a static box, as it does not take so much lines.

Reply:
ok